src/store/reducers/index.js

```diff
@@ -2,11 +2,13 @@ import { combineReducers } from 'redux';
 import { reducer as form } from 'redux-form';
 import products from './products';
 import product from './product';
+import signUp from './sign-up';
 
 const rootReducer = combineReducers({
   products,
   product,
   form,
+  signUp,
```

Comment:
A better name for this action/reducer/component/endpoint would be `login`. "signUp" is ambiguous and could refer to user sign up or  newsletter sign up or any other type of sign up.

Reply:
Is there a typo here?

Reply:
No typo here. I was just suggesting that you could rename the "signUp" reducer to "login"

Reply:
I am a bit hesitant to name it login. If I were to name it login, then what should I name my actual login reducer that handles the login state of the user?

Reply:
Great question! And, it's something that hasn't been answered definitively in the redux community... "How do you name reducers?"

Each project will have its own conventions based on the structure of the data and features within it. I typically name my reducers by the data that it represents. However, in all cases you want to make sure that the chosen name is not ambiguous.

It's really not a big deal at the moment but you may find the need to refactor so that it is more specific going forward.

Separately, I think I did make a typo... I intended to suggest renaming it to "register".

Reply:
Thank you for the explanation. I will go with register (y).

Reply:
👍